docs/admin/Nuxwdog.md

```diff
@@ -69,20 +69,20 @@ To disable Nuxwdog and use the plain password.conf
 
 ## Technical Implementation
 
-### `nuxwdog` python script
+### `pki-server-nuxwdog` python script
 
-[`nuxwdog`](base/server/scripts/nuxwdog) script is configured to run before the PKI server starts using [`systemd`
-unit file](base/server/scripts/nuxwdog). It uses `systemd-ask-password` to prompt the user for relevant passwords.
-The relevant passwords include `internal` and list of passwords defined in fields `cms.passwordlist` and
-`cms.tokenList` of every subsystem's `CS.cfg`. The passwords are stored on the Kernel Keyring provided by the
-`keyutils` package.
+[`pki-server-nuxwdog`](base/server/scripts/pki-server-nuxwdog) script is configured to run before the PKI server
+starts using [`systemd` unit file](base/server/share/lib/systemd/system/pki-tomcatd-nuxwdog@.service). It uses
+`systemd-ask-password` to prompt the user for relevant passwords. The relevant passwords include `internal` and
+list of passwords defined in fields `cms.passwordlist` and `cms.tokenList` of every subsystem's `CS.cfg`. The
+passwords are stored on the Kernel Keyring provided by the `keyutils` package.
```

Comment:
This is fine, but in the future try to put each sentence in a separate line to reduce the amount of changes due to reformatting.

Reply:
I believe the rpmbuild won't throw any errors for `.md` files for longer lines.